localization/localization-spanish/common-errors.md

```diff
@@ -0,0 +1,301 @@
+
+> Could not save configuration
+>
+> Firewall rules can't be changed without the required system permission.
+
+Se traducen a:
+
+> No se pudo guardar la configuración	
+>
+> Las reglas del cortafuegos no se pueden cambiar sin los permisos necesarios.
+
+## Tenga en cuenta las distinciones entre nombres simples y las aplicaciones con nombre propio
+
+A la hora de traducir, es importante  poder distinguir entre los nombres propios de las aplicaciones versus el uso común de las palabras que representan. 
+
+En particular, una aplicación, servicio o _plug_, al ser un nombre propio, debe ir siempre en su formato correcto, especificado en el glosario de términos y, si es posible, evitar referirse a la aplicación antecedida de un artículo (por ejemplo, es «Correo», no «el Correo»).
+
+Puede consultar un glosario de términos de todos los nombres propios de las aplicaciones básicas de elementary OS en el [glosario de términos](glossary.md).
+
+## Evite hacer traducciones absolutamente literales
+
+Las traducciones literales nunca son recomendadas. Es muy común querer traducir palabra por palabra la forma verbal del inglés para simular el flujo del idioma original. El español tiene una estructura diferente y utiliza modismos y estructuras verbales de manera distinta que el inglés.
+
+La sugerencia de esta guía es siempre leer la frase completa primero y entender el contexto detrás de ella para luego traducir. Trate de ver el todo de la frase y no sus palabras individuales para enviar el mismo mensaje, así no resulte necesariamente en una traducción literal.
+
+Es recomendable evitar _transliterar_ o _traducir palabra a palabra_, es decir, leer la primera palabra, traducirla, ir a la siguiente, traducirla, etcétera. Esto suele crear frases extrañas con una composición que no se siente natural al idioma español.
+
+### Ejemplo
+
+Consideremos esta frase:
+
+> This site uses cookies for incremental improvements. You may find the services function without them but at a reduced usability.
+
+La traducción literal tanto en palabras como gramatical resultaria en:
+
+> Este sitio usa _cookies_ para mejoras incrementales. Usted puede encontrar que los servicios funcionen sin ellas pero a una reducida usabilidad.
+
+Pero esto no se siente natural, tanto en vocablos como en gramática. Una traducción centrada más en la comprensión general de la frase sería algo como lo siguiente:
+
+> Este sitio utiliza _cookies_ para realizar mejoras continuas. Puede utilizar los servicios sin ellas pero con una usabilidad reducida.
+
+## Evite traducciones automáticas con traductores
+
+Es bastante común caer en la tentación de utilizar servicios de traducción como [el traductor de Google](https://translate.google.com) para obtener una traducción rápida.
+
+En principio, no tenemos objeciones sobre el uso de traductores para una base inicial, pero les suplicamos a las personas que hagan esto que revise _dos, tres o cuatro veces la traducción proporcionada por el traductor_.
```

Comment:
I'm actually not sure about the legal rights we may or may not have to use things translated by Google translate. So it's probably best to completely discourage this

Reply:
I have modified this bit to actually forbid the use of translators :+1: